mods/cef-titlebar-enabler-universal.wh.cpp

```diff
@@ -0,0 +1,259 @@
+// ==WindhawkMod==
+// @id              cef-titlebar-enabler-universal
+// @name            CEF/Spotify Titlebar Enabler
+// @description     Force native frames and title bars for CEF apps
+// @version         0.1
+// @author          Ingan121
+// @github          https://github.com/Ingan121
+// @twitter         https://twitter.com/Ingan121
```

Comment:
Please send me (https://twitter.com/m417z) a direct message with the following content:

> I attest that I'm the sole owner of both this Twitter account (https://twitter.com/Ingan121) and the following GitHub account: https://github.com/Ingan121

Reply:
Done